modules/nw-externalip-about.adoc

```diff
@@ -0,0 +1,184 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring_ingress_cluster_traffic/configuring-externalip.adoc
+
+[id="nw-externalip-about_{context}"]
+= About external IP addresses
+
+You can allow external traffic into the cluster by defining an *external IP* configuration.
+
+The configuration for external IP address blocks is defined by a Network custom resource (CR) named `cluster`.
+The Network custom resource (CR) is defined by the `Network.config.openshift.io` CustomResourceDefinition.
+
+The object is created during cluster installation by the Cluster Version Operator (CVO).
+
+// If it is a cloud deployment, what happens?
+// Is this configuration ignored, or does it break a cloud deployment?
+[IMPORTANT]
+====
+This feature is only supported in non-cloud deployments.
+For cloud deployments, use the load balancer services for automatic deployment of a cloud load balancer to target the service's endpoints.
+====
+
+Use of external IP in {product-title} is governed by the following parameters:
+
+// What happens to existing services if this is changed to another IP block?
+// What happens if policy rules change an address becomes rejected?
+
+* `spec.externalIP.autoAssignCIDRs` defines an IP address block used by the load balancer when choosing an external IP address for the service. {product-title} supports only a single IP address block for automatic assignment.
```

Comment:
If automatic assignment is enabled, then a Service of type LoadBalancer will automatically have an ExternalIP allocated for it.

This should not be used on cloud deployments.